app/assets/stylesheets/govuk_publishing_components/components/_share-links.scss

```diff
@@ -58,6 +58,17 @@ $share-button-height: 32px;
   }
 }
 
+.gem-c-share-links--columns {
+  .gem-c-share-links__list {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .gem-c-share-links__list-item {
+    flex: 0 0 200px;
```

Comment:
I think it might be worth adding a touch of padding-right to `.gem-c-share-links__list-item`, as otherwise you could end up with text running right up to the icon on the right:

<img width="310" alt="Screen Shot 2019-08-14 at 13 24 14" src="https://user-images.githubusercontent.com/1732331/63020740-d803a480-be96-11e9-87fd-2f30f17cabba.png">

Reply:
Definitely a contrived example and a rare use - I leave up to you whether it's worth fixing :)

Reply:
No, that's a good point. I've added a teeny bit of padding right (and padding left for rtl) and it seems to have fixed it. Thanks!